how-to-guides/ddp-training/scripts/train_ddp_multi_node_multiple_runs.py

```diff
@@ -67,6 +70,13 @@ def train(net, trainloader, run, rank, params):
             loss.backward()
             optimizer.step()
 
+            if i % 10 == 0:
+                for _, img in enumerate(images):
```

Comment:
```suggestion
                for img in images:
```

Can skip `enumerate` as index is not used